content/sigs/ux/index.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: sig
+title: "User Experience"
+section: sigs
+sigId: "ux"
+logo: /images/user.gif
+opengraph:
+  image: /images/user.gif
+tags:
+  - ux
+  - ui
+  - user_experience
+  - user_interface
+leads:
+- "jphartley"
+participants:
+- "oleg_nenashev"
+- "markewaite"
+links:
+  googlegroup: "jenkinsci-ux"
+  meetings: "https://docs.google.com/document/d/1xA6UN7ORv3OFYBC3Kw6y4mtqDekAjwYVNhsrH4PBwBk/edit?usp=sharing"
+overview: >
+  The User Experience SIG focuses on improving the Jenkins' user interface and user experience. The SIG seeks regular feedback from the community on the current UI and UX and new designs proposed by SIG members and others and will discuss alternatives, identify compromises, and test implementations to improve the Jenkins UI / UX
+---
+
+An overview of the Jenkins UX SIG documentation can be found by following this link:https://docs.google.com/document/d/1J3HsxYdNPDZpFzCz6HWGcIhsY3urOXOZmiMiGR1D-ew/edit?usp=sharing[link].
+
+=== Topics
+* Reviewing new UI proposals
+* Discussing UI frameworks
+* Discussing impact of UI changes to existing plugins
+
+=== Slack Channel
+The UX SIG uses Slack as the primary way to discuss any relevant topics. The channel used for this is #jenkins-ux-sig on the CDF Slack group. You can join the Slack channel via this link:https://join.slack.com/t/cdeliveryfdn/shared_invite/enQtODM2NDI1NDc0MzIxLTA1MDcxMzUyMGU2NWVlNmQwN2M1N2M4MWJjOWFkM2UzMDY0OWNkNjAzNzM0NzVkNjQ5M2NkMmY2MTRkMWY4MWY[link].
```

Comment:
Have preferred line per sentence in several other locations for easier reading and easier comparison of changes
```suggestion
The UX SIG uses Slack as the primary way to discuss any relevant topics. 
The channel used for this is #jenkins-ux-sig on the CDF Slack group. 
You can join the Slack channel via this link:https://join.slack.com/t/cdeliveryfdn/shared_invite/enQtODM2NDI1NDc0MzIxLTA1MDcxMzUyMGU2NWVlNmQwN2M1N2M4MWJjOWFkM2UzMDY0OWNkNjAzNzM0NzVkNjQ5M2NkMmY2MTRkMWY4MWY[link].
```